docker-compose.yml

```diff
@@ -17,8 +18,9 @@ services:
     build: .
     command: bash -l -c "npm install && npm start"
     ports:
-      - 5000:5000
-      - 5100:5100
+      - 5000:5000 #vite
+      - 5100:5100 #fastify
+      - 5555:5555 #prismaORM
     depends_on:
```

Comment:
I don't see the environment variable for the DB, I think we should add it so that things can get up and running without a `.env` file to start...

```
DATABASE_URL=postgresql://postgres@db/app
```

Reply:
@michaelyshih I still don't see this env var added in to the `docker-compose.yml` file, do you disagree with putting it in there (as opposed to requiring a `.env` file)?